test/concurrency/common_checker.h

```diff
@@ -30,6 +30,67 @@ auto Insert(Transaction *txn, BustubInstance &instance, int v1) -> void {
   ASSERT_EQ(ss.str(), "3,\n");
 }
 
+/// @brief Use this to write your own test cases, if needed
+/// @param v1_vec Elements to be insert into `t1:v1`, the size of which can either be 1, or same as @v2_vec
+/// @param v2_vec Elements to be insert into `t1:v2`, must not be empty
+/// @param flag When set to `true`, the size of @v1_vec should be exactly one, the default value is `false`
+/// Usage: Insert(txn, instance, {1, 2, 3}, {4, 5, 6}, flag = false);
+/// This overloaded function will then bound each pair from @v1_vec & @v2_vec as a insert value
+/// If flag is set to true, v1_vec should contain exactly one element,
+/// The insert pair will be {v1_vec[0], v2_vec[0]}, {v1_vec[0], v2_vec[1]} ...
+auto Insert(Transaction *txn, BustubInstance &instance, const std::vector<int> &v1_vec, const std::vector<int> &v2_vec,
+            bool flag = false) -> void {
+  // Check if the input vectors have the same size.
+  if (!flag) {
+    if (v1_vec.size() != v2_vec.size()) {
+      fmt::print(stderr, "Vectors @v1_vec and @v2_vec must have the same size\n");
+      return;
+    }
+    if (v1_vec.empty()) {
+      fmt::print(stderr, "Input vectors must not be empty\n");
+      return;
+    }
+  } else {
+    if (v1_vec.size() != 1) {
+      fmt::print(stderr, "Size of @v1_vec should be exactly 1 if flag is set to `true`\n");
+      return;
+    }
+    if (v2_vec.empty()) {
+      fmt::print(stderr, "Input vectors must not be empty\n");
+      return;
+    }
+  }
+
+  std::stringstream ss;
+  auto writer = bustub::SimpleStreamWriter(ss, true, ",");
+
+  assert(v1_vec.size() == 1);
+  std::string val_str{};
+  if (!flag) {
+    for (size_t i = 0; i < v1_vec.size(); ++i) {
+      val_str += fmt::format("({}, {}), ", v1_vec[i], v2_vec[i]);
+    }
+  } else {
+    for (const auto &v2 : v2_vec) {
+      val_str += fmt::format("({}, {}), ", v1_vec[0], v2);
```

Comment:
does the parser work with trailing comma?

Reply:
I removed the trailing comma and space on line 81 & 82, so I guessed it should be good.
But though, I tested `libpg_query` locally and found out it supported neither parsing trailing comma nor (trailing comma + extra space)
![image](https://github.com/cmu-db/bustub/assets/91381303/c024af0b-236f-4e48-addc-20584bdea9a1)